docs/conf.py

```diff
@@ -22,114 +16,52 @@
 
 import distro  # noqa: E402
 
-# -- General configuration ------------------------------------------------
+# -- Project information -----------------------------------------------------
+
+project = "distro"
+copyright = "2015,2016, Nir Cohen, Andreas Maier"
+author = "Nir Cohen, Andreas Maier"
```

Comment:
these values need to be updated.

Reply:
Let's dynamically generate the year with `datetime.date.today().year` or the like.

Reply:
Thanks @sethmlarson I applied your suggestion in the latest revision.